client/rest-high-level/src/main/java/org/elasticsearch/client/security/user/privileges/GlobalPrivileges.java

```diff
@@ -37,64 +40,79 @@
 
 /**
  * Represents global privileges. "Global Privilege" is a mantra for granular
- * privileges over applications. {@code ApplicationResourcePrivileges} model
- * application privileges over resources. This models user privileges over
- * applications. Every client is responsible to manage the applications as well
- * as the privileges for them.
+ * generic cluster privileges. These privileges are organized into categories.
+ * Elasticsearch defines the set of categories. Under each category there are
+ * operations that are under the clients jurisdiction. The privilege is hence
```

Comment:
This is not true.
The definition 
```
"application": { "manage": { "applications": [ "kibana-*" ] } }
```
Instructs Elasticsearch that this role is allowed to manage application privileges under the application names "kibana-*".
It's not an instruction for Kibana, it's enforced by Elasticsearch.

Reply:
I was wrong, I got carried astray by to much abstraction.
This is now:
```
Constructs privileges under a specific {@code category} and for some
{@code operation}. The privilege definition is flexible, it is a {@code Map},
and the semantics is bound to the {@code category} and {@code operation}.
```